### PR TITLE
Update content-style-guide.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding this official stance to the style guide to ensure correct terminology is used

### CODE REVIEW

File: contributing/content-style-guide.md [hunk 1]
```diff
@@ -307,10 +307,10 @@ A numbered list following an H2 heading:
 ## Tables
 Tables provide a great way to present complex information and can help the content be more scannable for users, too.
 
-There are many ways to construct a table, like row spanning and cell splitting. Make sure the content is clear, concise, and presents well on the web page (like avoid awkward word wrapping).
+There are many ways to construct a table, like row spanning and cell splitting. Make sure the content is clear, concise, and presented well on the web page (like avoiding awkward word wrapping).
```
Comment (matthewshaver) [inline contributing/content-style-guide.md]: We should avoid using "like" to present examples.
```suggestion
There are many ways to construct a table, such as row spanning and cell splitting. The content should be clear, concise, and presented well on the web page (for example, avoid awkward word wrapping).
```

File: contributing/content-style-guide.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ You can configure a card in 2, 3, 4, or 5-column grids. To maintain a good user
 
 There won't be many instances where you need to display 4 or 5 cards on the docs site. While we recommend you use 2 or 3-column grids, you can use 4 or 5-column grids in the following scenarios:
 
-- For cards that contain little text and limited to under 15 words. (This is to make sure the text isn't squished)
+- For cards that contain little text and are limited to under 15 words. (This is to make sure the text isn't squished)
```
Comment (matthewshaver) [inline contributing/content-style-guide.md]:
```suggestion
- For cards that contain little text and are limited to 15 words or less. This is to make sure the text isn't squished.
```

File: contributing/content-style-guide.md [hunk 3]
```diff
@@ -488,9 +488,14 @@ Avoid ending a sentence with a preposition unless the rewritten sentence would s
 
 Product names, trademarks, services, and tools should be written as proper nouns, unless otherwise specified by the company or trademark owner.
 
+As of October 2023, avoid using "dbt CLI" or "CLI" terminology. dbt officially provides two command line tools for running dbt commands:
+
+- [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) &mdash; This tool allows you to develop locally and execute dbt commands against your dbt Cloud development environment from your local command line.
+- [dbt Core](https://github.com/dbt-labs/dbt-core) &mdash; This open-source tool is designed for local installation, enabling you to use dbt Core on the command line and communicate with databases through adapters.
```
Comment (matthewshaver) [inline contributing/content-style-guide.md:496]: Either this should say dbt CLI, or we need to go through our docs and change every instance (and page) with `dbt CLI` (of which there are a bunch) to `dbt Core`. Do you know what the preference is from a branding perspective? I thought the Core CLI was the "dbt CLI"

File: contributing/content-style-guide.md (hunk 1 above)
Comment (mirnawong1) [reply contributing/content-style-guide.md]: i hear you @matthewshaver  might be worth discussing with the docs team bc the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/2db0204a02fdf9150732662b7369ba50c7a6f65f/contributing/content-style-guide.md#latin-abbreviations) itself suggests using 'like' (and i was also told to use 'like', 'such as', 'for example', as opposed to 'eg' when i started).

Screenshot 2023-10-27 at 11 48 19

File: contributing/content-style-guide.md [hunk 4]
```diff
@@ -488,9 +488,14 @@ Avoid ending a sentence with a preposition unless the rewritten sentence would s
 
 Product names, trademarks, services, and tools should be written as proper nouns, unless otherwise specified by the company or trademark owner.
 
+As of October 2023, avoid using "dbt CLI" or "CLI" terminology. dbt officially provides two command line tools for running dbt commands:
```
Comment (mirnawong1) [inline contributing/content-style-guide.md]:
```suggestion
As of October 2023, avoid using "dbt CLI" or "CLI" terminology when referring to the dbt Cloud CLI or dbt Core.  However, if referring to the command line as a tool, CLI is acceptable.

dbt officially provides two command line tools for running dbt commands:
```

**Comment by mirnawong1** [comment]
hey @matthewshaver , thanks so much for this review! is this good to merge or are there any pending items for me to resolve?

File: contributing/content-style-guide.md (hunk 3 above)
Comment (runleonarun) [reply contributing/content-style-guide.md:496]: Done by Mirna and Matt!

**Review by runleonarun — APPROVED** [review]
LGTM!

**Comment by mirnawong1** [comment]
thanks so much for merging @runleonarun !

**Pull request merged**